### ISSUE TITLE
Question: Would you be open to discuss ways of making bettercode reviews a bit more visible and or easy to find? 

### ISSUE THREAD

**Issue by roalcantara** [vaidehijoshi/better-code-reviews#7]
Hi @vaidehijoshi,

Firstly, thank you so much for such an initiative. This is something that I miss deeply in our community.

## TL;DR:

Would you be open to discuss ways of making [bettercode reviews](http://www.bettercode.reviews/) a bit more visible and or easy to find? Perhaps, a sort of [specification](https://semver.org/) or even an [opencollective](https://opencollective.com/) sort of initiative?

## Now, the too long (aka skippable) version:

I also love this topic, and I've experienced both their benefits and drawbacks of its lack.

Naturally, as an average-developer who does not want to preach each new team member, I've been trying to compile the best resources that I could find - a sort of [guideline](https://github.com/roalcantara/trying-to-suck-less/blob/main/README.md) if you will.

Such was my surprise when I come across the [bettercode reviews](http://www.bettercode.reviews/).

Have I seen it before, it would have saved me a few grey hairs, I reckon. 😅

As my hair is beyond any salvation, I could not help wondering again: how come I could not find it before. After all, it is such great content - and I could not think of a better domain.

My hypothesis so far is that I might probably not be the best google user. Which I must shamefully agree to some extend. 😅

### But that has led me to the second point

Would it be possible that others average-developers (like myself) have also experienced any difficulty in finding [bettercode reviews](http://www.bettercode.reviews/) - or even not being able to find it at all?

Indeed, I have no idea on the [bettercode reviews](http://www.bettercode.reviews/)'s statistics, so I could only guess, - which is something that I'm not very fond.

Yet, I could not help [noticing](https://lighthouse-dot-webdotdevsite.appspot.com//lh/html?url=http%3A%2F%2Fwww.bettercode.reviews%2F) that it might be a few tweaks that could perhaps have a positive impact on the site.

### Which, finally, brings me to my (I promise) last question:

Would you be open to discuss ways of making [bettercode reviews](http://www.bettercode.reviews/) a bit more visible and or easy to find? Perhaps, a sort of [specification](https://semver.org/) or even an [opencollective](https://opencollective.com/) sort of initiative?

I apologise for opening the issue. I wasn't sure of opening a Pull Request before knowing your opinion.

That's pretty much it.

I look forward to hearing your thoughts.

And let **BettercodeReviews** sparkle, shimmer, sing, swing, swirl, shock, and sizzle!

(I must admit having read this piece on a blog, and I could not miss the opportunity of using it, I apologise! 😅)

If you really could bear with me until here, I have no option other than thank you so much for your and monk-like-patience and apologise for taking your time.

### PS: Disclaimer
As you might have noticed, English is not my first language.
And even the best efforts of my Grammarly could not help me fail miserably to convey some bad written idea. That said, please, if I happened to have said something that sounds weird or stupid, I beg you to consider ignorance over bad intentions.
